SilverSolver/assignment1/Q1.py

```diff
@@ -0,0 +1,49 @@
+def are_words_anagram(s1, s2, case_sensitive=False, return_dicts=False):
+    alphabet = "abcdefghijklmnopqrstuvwxyz"
```

Comment:
defining the alphabet yourself is prone to mistakes, and is likely to diverge from other alphabet definition.
Can you use the character classes "non-whitespace" found in the re module? The symbol for the character class is \W
https://docs.python.org/3/library/re.html